pkg/webhook/v1beta1/pod/inject_webhook.go

```diff
@@ -322,7 +330,7 @@ func (s *SidecarInjector) getMetricsCollectorArgs(trial *trialsv1beta1.Trial, me
 		suggestion := &suggestionsv1beta1.Suggestion{}
 		err := s.client.Get(context.TODO(), apitypes.NamespacedName{Name: suggestionName, Namespace: trial.Namespace}, suggestion)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("%w: %s", errInvalidSuggestionName, err.Error())
```

Comment:
```suggestion
			return nil, fmt.Errorf("%w: %w", errInvalidSuggestionName, err)
```

Reply:
This has been fixed in this [commit](https://github.com/kubeflow/katib/pull/2289/commits/e0518aca1cb4b33693b617285abd049f1d5e0e23).